From 93fb8a9e152d073f609dfdd9f108a76df97fabba Mon Sep 17 00:00:00 2001
From: Hiep Pham <hiep.pham.zy@renesas.com>
Date: Tue, 10 Feb 2026 13:48:30 +0700
Subject: [PATCH 3/7] arm64: defconfig: enable LVDS and IT6263 LVSD to HDMI
 bridge

Enable LVDS and IT6263 LVSD to HDMI bridge configs for RZ/G3E.

Signed-off-by: hienhuynh <hien.huynh.px@renesas.com>
Signed-off-by: Hiep Pham <hiep.pham.zy@renesas.com>
---
 arch/arm64/configs/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 6602ce7459281..964ea5e29e5a9 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -244,6 +244,9 @@ CONFIG_DRM_RCAR_DW_HDMI=y
 CONFIG_DRM_RCAR_MIPI_DSI=y
 CONFIG_DRM_RZG2L_MIPI_DSI=y
 CONFIG_DRM_RZG2L_DU=y
+CONFIG_DRM_RZG3E_USE_LVDS=y
+CONFIG_DRM_RZG3E_LVDS=y
+CONFIG_DRM_ITE_IT6263=y
 CONFIG_DRM_PANEL_LVDS=y
 CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_DRM_PANEL_EDP=y
-- 
2.25.1

